maskit/examples/load_iris.py

```diff
@@ -0,0 +1,36 @@
+from pennylane import numpy as np
+from sklearn import datasets
+from maskit.examples.utils import one_hot
+
+np.random.seed(42)
+
+MAX_TRAIN_SAMPLES = 150
+
+
+def load_iris(wires, params):
+    train_size = MAX_TRAIN_SAMPLES
+    if "train_size" in params:
+        train_size = min(params["train_size"], MAX_TRAIN_SAMPLES)
+    data, target = datasets.load_iris(return_X_y=True)
+    target = target.reshape((150, 1))
```

Comment:
If the ``150`` is here because it's the ``MAX_TRAIN_SAMPLES``, using the constant makes it much clearer what is going on.
```suggestion
    target = target.reshape((MAX_TRAIN_SAMPLES, 1))
```